README.md

```diff
@@ -443,7 +443,7 @@ var converter = new showdown.Converter({ extensions: ['twitter'] });
 ```js
 var showdown    = require('showdown'),
     myExtension = require('myExtension'),
-    converter = new showdown.Converter({ extensions: ['myExtension'] });
+    converter = new showdown.Converter({ extensions: [myExtension] });
```

Comment:
This won't work.
When adding a custom extension, you also need to pass a name for your extension like this:
`{ extensions: [ 'myExtension', myExtension ] }`

This README file is outdated, and we're in the middle of deprecating it.
Please check the [Extensions documentation](https://showdownjs.com/docs/extensions/) for more details.

Sorry for the inconvenience.